src/CoreServices/FSEvents.cs

```diff
@@ -199,6 +199,30 @@ public FSEventStream (string [] pathsToWatch, TimeSpan latency, FSEventStreamCre
 		{
 		}
 
+		[DllImport (Constants.CoreServicesLibrary)]
+		static extern IntPtr FSEventStreamCreateRelativeToDevice (IntPtr allocator, FSEventStreamCallback callback, 
+		IntPtr context, ulong deviceToWatch, IntPtr pathsToWatchRelativeToDevice, 
+		ulong sinceWhen, double latency, FSEventStreamCreateFlags flags);
+
+		public FSEventStream (CFAllocator allocator, ulong deviceToWatch, NSArray pathsToWatchRelativeToDevice, 
```

Comment:
The `allocator` is optional, so you can provide an overload without it, that just calls this version:

```csharp
public FSEventStream (ulong deviceToWatch, NSArray pathsToWatchRelativeToDevice, ulong sinceWhen, TimeSpan latency, FSEventStreamCreateFlags flags)
    : this (null, deviceToWatch, pathsToWatchRelativeToDevice, sinceWhen, latency, flags)
{
}
```